jfix-armeria-facade/src/main/kotlin/ru/fix/armeria/facade/webclient/HttpClientBuilderApi.kt

```diff
@@ -23,6 +23,8 @@ interface BaseHttpClientBuilder<out HttpClientBuilderT : BaseHttpClientBuilder<H
 
     fun setClientName(clientName: String): HttpClientBuilderT
 
+    fun setIgnoreEndpoint(ignore: Boolean): HttpClientBuilderT
```

Comment:
I think is is a bad idea to force clients to explicitly say when they do not want to specify some of builder params. It is not a fluent API when you cannot just do not specify some param, but you must explicitly say "please, ignore that I have not specified some param". 

It would be better to just make some parameter optional and to not fail build when this parameter is not specified